.changeset/silly-dancers-beg.md

```diff
@@ -0,0 +1,5 @@
+---
+"geohub": minor
+---
+
+feat: adjust AnalyzeBivariate to match other UI elements
```

Comment:
Could you edit existing changeset file instead of adding new file?

Existing changeset for develop-ibm is here

https://github.com/UNDP-Data/geohub/blob/develop-ibm/.changeset/cuddly-bulldogs-leave.md

we only need a changeset file for develop-ibm branch only.

Reply:
Hi Jin, I've pushed a commit moving this to the mentioned changeset file. Thanks!